src/libraries/System.Globalization/tests/CultureInfo/CultureInfoDateTimeFormat.cs

```diff
@@ -65,5 +65,34 @@ public void DateTimeFormat_Set_Invalid()
             AssertExtensions.Throws<ArgumentNullException>("value", () => new CultureInfo("en-US").DateTimeFormat = null); // Value is null
             Assert.Throws<InvalidOperationException>(() => CultureInfo.InvariantCulture.DateTimeFormat = new DateTimeFormatInfo()); // DateTimeFormatInfo.InvariantInfo is read only
         }
+
+        public static IEnumerable<object[]> DateTimeFormat_En_Locales_ShortDatePattern_TestData()
+        {
+            yield return new object[] { "en-AS", "M/d/yyyy" };
+            yield return new object[] { "en-AU", "d/M/yyyy" };
+            yield return new object[] { "en-BI", "M/d/yyyy" };
+            yield return new object[] { "en-CA", "yyyy-MM-dd" };
+            yield return new object[] { "en-GU", "M/d/yyyy" };
+            yield return new object[] { "en-HK", "d/M/yyyy" };
+            yield return new object[] { "en-MH", "M/d/yyyy" };
+            yield return new object[] { "en-MP", "M/d/yyyy" };
+            yield return new object[] { "en-NZ", "d/MM/yyyy" };
+            yield return new object[] { "en-PR", "M/d/yyyy" };
+            yield return new object[] { "en-SE", "yyyy-MM-dd" };
+            yield return new object[] { "en-SG", "d/M/yyyy" };
+            yield return new object[] { "en-UM", "M/d/yyyy" };
+            yield return new object[] { "en-US", "M/d/yyyy" };
+            yield return new object[] { "en-VI", "M/d/yyyy" };
+            yield return new object[] { "en-ZA", "yyyy/MM/dd" };
+            yield return new object[] { "en-ZW", "d/M/yyyy" };
+        }
+
+        [Theory]
+        [MemberData(nameof(DateTimeFormat_En_Locales_ShortDatePattern_TestData))]
+        public void DateTimeFormat_En_Locales_ShortDatePattern(string locale, string shortDatePattern)
+        {
+            var cultureInfo = new CultureInfo(locale);
+            Assert.Equal(shortDatePattern, culture.DateTimeFormat.ShortDatePattern);
```

Comment:
You meant, `cultureInfo.DateTimeFormat...`? 
```suggestion
            Assert.Equal(shortDatePattern, cultureInfo.DateTimeFormat.ShortDatePattern);
```

Reply:
oops 🙂